examples/gnrc_border_router/README.md

```diff
@@ -217,7 +217,7 @@ USEMODULE += gnrc_slip
 ## Configuration
 
 In order to connect a RIOT 6LoWPAN border router over SLIP you run a small
-program called tunslip (imported from Contiki) [2] on the Linux host. The
+program called tunslip6 (imported from Contiki) [2] on the Linux host. The
```

Comment:
I think the doc should reflect that ;-). Also that either tool is only needed if ethos or slip is used. For an ethernet device no such tool is needed.